pkg/crc/systemd/systemd.go

```diff
@@ -0,0 +1,90 @@
+package systemd
+
+import (
+	"fmt"
+
+
+	. "github.com/code-ready/crc/pkg/os"
```

Comment:
Do explicit mention the pkg here like for this one you can use `crcos`, it will help to track which functions coming from which package.

Reply:
I hate this too. Noticed the pre-flights did the same and this causes "Goto Definition" not to work correctly.

Reply:
Will also fix this in the preflight code...